manage_cnpg.yml

```diff
@@ -0,0 +1,98 @@
+        # drop-db, drop-schema, drop-role
+        # execute-sql-script
+        # apply-manifest, remove-manifest
+        # configure-memory, configure-pg-hba
+        # scale-replicas
+        # get-superuser-secret, add-postgres-expose-service, drop-postgres-expose-service
+        # add-grafana-expose-service, drop-grafana-expose-service
+        # promote-pod
+        # reload-cluster, restart-cluster, restart-pod
+        # maintenance-set, maintenance-unset, status-cluster
+        cnpg_task: 
+        #- drop-schema
+        #- drop-role
+        #- drop-schema
+        #- add-db
+        #- add-role
+        #- drop-db
+        #- scale-replicas
+        #- get-superuser-secret
+        #- add-postgres-expose-service
+        #- add-grafana-expose-service
+        #- drop-postgres-expose-service
+        #- drop-grafana-expose-service
+        #- promote-pod
+        #- reload-cluster
+        #- restart-cluster
+        #- restart-pod
+        #- maintenance-set
+        #- maintenance-unset
+        - status-cluster
+        # CloudNativePG Namespace
+        cnpg_namespace: default
+        # CloudNativePG Cluster Name
+        cluster_name: cnp-sandbox
+        # CloudNativePG Pod Name
+        pod_name: cnp-sandbox-1
+        # Databases
+        db_name: edb
+        # Schemas
+        schema_name: edb_schema
+        # Roles
+        db_role: edb_role
+        db_role_password: admin
+        # SQL Script
+        sql_script: DROP DATABASE {{ db_name }};
+        # Manifest Filename
+        manifest_filename: configure_memory.yml
+        # Scale Replicas Number
+        scale_replicas_to: 5
+        # Scale Manifest Filename
+        #scale_manifest_filename: roles/manage_cnpg/files/cluster-example.yml
+        scale_manifest_filename: roles/manage_cnpg/files/cnp-sandbox.yml
+        # Cluster SuperUser Name
+        cluster_superuser_name: cnp-sandbox-superuser
+        # Cluster Read Write Service Name
+        cluster_rw_service_name: cnp-sandbox-rw
+        # Cluster External Ingress Service Name
+        cluster_external_ingress_service_name: cnp-sandbox-external-ingress
+        # Cluster External Source Port
+        cluster_external_ingress_source_port: 5432
+        # Cluster External Target Port
+        cluster_external_ingress_target_port: 5432
+        # Cluster Grafana Deployment Name
+        cluster_grafana_deployment_name: cnpg-sandbox-grafana
+        # Grafana External Ingress Service Name
+        cluster_grafana_external_ingress_service_name: grafana-external-ingress
+        # Cluster Grafana External Source Port
+        cluster_grafana_external_ingress_source_port: 3000
+        # Cluster Grafana External Target Port
+        cluster_grafana_external_ingress_target_port: 3000
+        # Pod to promote
+        pod_to_promote: 1
+        # CloudNativePG Pod Name to Restart
+        pod_name_to_restart: cnp-sandbox-2
+
+  roles:
+    - role: manage_cnpg
+      when: cnpg_task is defined
```

Comment:
This file also seems to appear in `/playbook-examples/manage_cnpg/manage_cnpg.yml`. The directory `/playbook-examples/manage_cnp/` also still exists. Please remove the `manage_cnp` directory from the `playbook-examples` and have this `manage_cnpg.yml` only exist in the `playbook-examples/manage_cnpg/` directory.

Reply:
have removed unnecessary file